### PR TITLE
Corrected readme.md and optimised images through ImgBot.

### PR DESCRIPTION
**Pull request by Rohan-cod**

### CODE REVIEW

**Comment by bd808** [comment]
@Rohan-cod I did not notice that you had an open pull request here before I did some major changes in this fork of the upstream project. The "toolforge" branch is now our deployed branch. I have reset the "master" branch back to being a clean copy of the upstream's master. Your pull request will need manual rebasing to be reviewable.

I have doubts about the commit in your patch chain that bumps express from 2.5.10 to 4.17.1 with no code changes other than the library version. I find it fairly unlikely that in 2 major version bumps there were no API changes in the express components used in this project. I'm also not sure that it is a great idea for us to make large changes like that in our fork. Contributing changes like that to the upstream project seems like a better idea. We should then be able to bring the upstream changes down into this repo and rebase our local Toolforge specific changes on top of them.

**Comment by Rohan-cod** [comment]
So Sir, should i remove the commit for the dependency changes and keep the rest of the commits?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+
+# Hatjitsu
+
+Create disposable online [Planning Poker](http://en.wikipedia.org/wiki/Planning_poker) rooms for quick and easy estimations.
+
+## Features
+=======
```
Comment (bd808) [inline README.md]: This file looks like you had merge conflicts and resolved them by keeping both sets of changes? There is certainly a lot of duplicated information in the README now.

File: app/css/app.css [hunk 2]
```diff
@@ -87,11 +87,11 @@ a:focus {
     height: 56px;
     width: 155px;
     text-indent: -1000px;
-    background-image: url(/img/title.png);
+    background-image: url(../img/title.png);
```
Comment (bd808) [inline app/css/app.css]: This is reverting changes that I made recently.

File: app/index.ejs [hunk 3]
```diff
@@ -17,41 +17,28 @@
...
-  </script>
+  <script src="lib/modernizr.custom.07116.js"></script>
+  <script src="js/testing.js"></script>
 </head>
 <body>
   <div class="body bg{{bodyState}}" ng-controller="MainCtrl">
     <div class="container">
       <header class="header">
-        <a href="/" title="Back to the Lobby">
+        <a href="/hatjitsu/" title="Back to the Lobby">
```
Comment (bd808) [inline app/index.ejs]: Another revert of recent changes.

File: package.json [hunk 4]
```diff
@@ -6,15 +6,11 @@
...
-  "scripts": {
-    "start": "server.js"
-  },
   "engines": {
     "node": "0.8.x"
   },
   "dependencies": {
-    "express": "2.5.10",
-    "express-cdn": "0.0.3",
+    "express": "4.17.1",
```
Comment (bd808) [inline package.json]: This is a major API change for express. Bumping the package version here without other code changes will break the entire application. This kind of major change should be made upstream and not in this fork.

**Review by bd808 — CHANGES_REQUESTED** [review]
> So Sir, should i remove the commit for the dependency changes and keep the rest of the commits?

The library version bump almost certainly needs to be removed. The use of a merge commit to catch up with the new HEAD seems to have done some unhelpful things to your pull request as well.

It really might be easier for you to back up and redo the imagebot and readme changes manually than trying to fix the git history here. But I also wonder if there is any real value in either set of changes? The README changes from the first patch are visual linting more than anything else. The image compression changes seem to be the thing that would be most valuable.

**Comment by Rohan-cod** [comment]
@bd808, I have made the necessary changes. Please have a look.

**Comment by bd808** [comment]
> @bd808, I have made the necessary changes. Please have a look.

The state of the collective pull request is still quite messy. If you look at the "Files changed" view you can see that 19 files are changed. Of those 19 files, I believe only the 8 files under  app/img are constructive edits. These are the changes from 3901c57 which were authored by @ImgBotApp.

I would accept a pull request (this one or a fresh start) that includes _only_ this commit. To salvage this pull request, I believe this should work:
```shell
$ cd location_of_your_clone

# Make sure we are on the "master" branch
$ git checkout master

# Fetch the current state of this upstream repo into your working copy's index
$ git add remote upstream https://github.com/toolforge/tool-hatjitsu
$ git fetch upstream

# Replace the contents of the current branch in your working copy
# with contents of the "toolforge" branch from this repo
$ git reset --hard upstream/toolforge

# Copy the 3901c57 commit that will still be in your working copy's index
# to the HEAD of your branch
$ git cherry-pick 3901c57a8dab15fdfc6f89ecea7644120e728c43

# Publish these changes to the "master" branch of your GitHub fork
# which will by extension also update this pull request.
$ git push --force
```

Once those steps are done, this pull request should _only_ show one commit (3901c57, although I am fairly certain it will have a new hash) and 8 files changed (the images in app/img).

**Comment by Rohan-cod** [comment]
Thanks for the detailed explanation @bd808 😄

**Comment by bd808** [comment]
Thanks @Rohan-cod!

**Comment by Rohan-cod** [comment]
My pleasure 😄.
Thanks for all the support @bd808.

**Pull request merged**